github/Repository.py

```diff
@@ -2788,6 +2789,33 @@ def get_readme(self, ref=github.GithubObject.NotSet):
             self._requester, headers, data, completed=True
         )
 
+    def get_self_hosted_runner(self, runner_id):
```

Comment:
I think this method and the other two below have not been added to Repository.pyi for type hints.